daemons/scheduled-ci-build/app.py

```diff
@@ -14,33 +15,39 @@
 app = domovoi.Domovoi(configure_logs=False)
 
 
-def define_test_type(test_type: str, envvar: str, schedule_expression: str):
-    @app.scheduled_function(schedule_expression)
-    def integration_test(event, context):
+def define_build(branch_name: str, build_type: str, schedule_expression: str, **env):
+    def f(event, context):
         travis_token = os.environ["TRAVIS_TOKEN"]
         rule_name = event["resources"][0].split(":")[-1]
         body = {
             'request': {
-                'branch': "master",
-                'message': f"{test_type} test started by {context.function_name} from {rule_name}",
+                'branch': branch_name,
+                'message': f"{build_type} f started by {context.function_name} from {rule_name}",
```

Comment:
did you mean to replace `test` with `f` here?

Reply:
Oops. Will fix.